internal/profile/profile_json.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package profile
+
+import (
+	"encoding/json"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/version"
+)
+
+// Metadata stores the data associated with a given profile
+type Metadata struct {
```

Comment:
I could be wrong but I think this type can be unexported? AFAICT it's not being used from anywhere outside the `profile` package.

Reply:
Scratch that, looks like it's needed in `cmd/profiles.go`.